### PR TITLE
arvo: language-server desk support

### PR DESCRIPTION
**Pull request by ryjm**
Looks in all desks when attempting to build a file and picks the first match.

### CODE REVIEW

File: pkg/arvo/app/language-server.hoon [hunk 1]
```diff
@@ -217,7 +218,7 @@
   :_  (give-rpc-notification (get-diagnostics uri))
   :*
     %pass
-    /commit
+    /commit˘
```
Comment (liam-fitzgerald) [inline pkg/arvo/app/language-server.hoon:221]: ???

File: pkg/arvo/app/language-server.hoon [hunk 2]
```diff
@@ -240,18 +241,29 @@
...
   ?~  p.gift
-    [~ state]
+    [[~ state] %base]
   =.  builds
     (~(put by builds) uri q.r.u.p.gift)
   =.  ford-diagnostics
     (~(del by ford-diagnostics) uri)
-  =+  .^(=open:clay %cs /(scot %p our.bow)/home/(scot %da now.bow)/open/foo)
-  =/  =type  -:(open (uri-to-path:build uri))
+  =/  bek  byk.bow
```
Comment (liam-fitzgerald) [inline pkg/arvo/app/language-server.hoon:253]: This will have the wrong case, you'll want
```hoon
=/  bek  byk.bow(r da+now.bow)
```

**Review by liam-fitzgerald — CHANGES_REQUESTED** [review]
Some nits, general idea looks good. Wondering if you have any ideas for the direction of cross-desk support in the LSP?

File: pkg/arvo/app/language-server.hoon (hunk 1 above)
Comment (ryjm) [reply pkg/arvo/app/language-server.hoon:221]: what it's cute

**Review by liam-fitzgerald — APPROVED** [review]

**Pull request merged**